Tests/SwiftStdlibTests/KeyedDecodingContainerExtensionsTests.swift

```diff
@@ -31,41 +31,45 @@ final class KeyedDecodingContainerTests: XCTestCase {
     func testDecodeBoolAsIntOrStringDataAsIntSuccessful() {
         let isPlayingAndIsFullScreenAsInt = #"{"isPlaying": 1, "isFullScreen": 0}"#
         let data = mockJsonData(from: isPlayingAndIsFullScreenAsInt)
-        let video = try! JSONDecoder().decode(Video.self, from: data)
-        XCTAssertTrue(video.isPlaying)
-        XCTAssertEqual(video.isFullScreen, false)
+        let video = try? JSONDecoder().decode(Video.self, from: data)
```

Comment:
```suggestion
        let video = try! JSONDecoder().decode(Video.self, from: data) // swiftlint:disable:this force_try
```

Reply:
Hello @guykogus 
Yes, the problem is that we cannot use `try!` with swiftlint.